books/free-programming-books-no.md

```diff
@@ -7,3 +7,4 @@
 
 * [LaTeX for nybegynnere](https://www.mn.uio.no/ifi/tjenester/it/hjelp/latex/latex-for-nybegynnere.pdf) - Dag Langmyhr (PDF)
 * [LaTeX for viderekomne](https://www.mn.uio.no/ifi/tjenester/it/hjelp/latex/latex-videre.pdf) - Dag Langmyhr (PDF)
+* [Lokal guide til farger i LaTeX](https://www.mn.uio.no/ifi/tjenester/it/hjelp/latex/farger.pdf) - Dag Langmyhr (PDF)
```

Comment:
All these links look really similar. I wonder if there is a more generic link we could use that captures all of these. If not, this looks good to me!

Reply:
This address -> https://www.mn.uio.no/ifi/tjenester/it/hjelp/latex/ <- does have a bunch of latex related information, unfortunately it is a mix of Norwegian and English and I found at least one dead link and another that leads to an ftp server. I don't know of a good way to deal with that.